backend/tests/controllers/promoter/test_promoter.py

```diff
@@ -116,39 +116,44 @@ def test_run_to_move_artefact_deb(
         repo="main",
         created_at=datetime.utcnow(),
     )
-    create_artefact(
+    artefact2 = create_artefact(
         db_session,
         "proposed",
-        name="linux-generic",
-        version="5.19.0.43.38",
+        name="linux-oem-22_04a",
+        version="6.1.0.1028.29",
```

Comment:
can you please add tests with debname having `_` and `.` to test your changes?

Reply:
I couldn't find a real example of such a case, so I added a contrived one